frontend/src/components/elements/Text/Text.tsx

```diff
@@ -21,15 +21,18 @@ import classNames from "classnames"
 import ReactJson from "react-json-view"
 import ReactMarkdown from "react-markdown"
 import { Map as ImmutableMap } from "immutable"
-import { Text as TextProto } from "autogen/proto"
-
 // Ignoring typeScript for this module as it has no ts support
 // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
 // @ts-ignore
```

Comment:
I didn't know this worked! Now I'm not sure my solution here is the best one:
https://github.com/tvst/streamlit/blob/4ab640037bc6a968886ce93abb0897d0b77f0755/frontend/src/declarations.d.ts